src/test/db/multitenant/00-storage-schema.sql

```diff
@@ -0,0 +1 @@
+CREATE SCHEMA IF NOT EXISTS storage;
```

Comment:
ah this should be in /migrations/multitenant/0001-initialmigration.sql (or just use public schema)

Reply:
Switched to public schema.